app/models/custom_filter.rb

```diff
@@ -114,6 +114,8 @@ def invalidate_cache!
     Rails.cache.delete("filters:v3:#{account_id}")
     redis.publish("timeline:#{account_id}", Oj.dump(event: :filters_changed))
     redis.publish("timeline:system:#{account_id}", Oj.dump(event: :filters_changed))
+
+    redis.publish('system', Oj.dump(event: :filters_changed, account_id: account_id))
```

Comment:
This may look a little confusing, but here we've both a "system channel" for the filters change (meaning that we need to refresh them in memory) and an events topic for the client to receive notification of the filters changing.

The final state of this code would only have two publishes: one for the account and one to system channel.